java/libs/src/main/java/org/physical_web/collection/PhysicalWebCollection.java

```diff
@@ -52,14 +53,18 @@ public PhysicalWebCollection() {
     mIconUrlToIconMap = new HashMap<>();
     mPendingBroadcastUrls = new HashSet<>();
     mPendingIconUrls = new HashSet<>();
+    mFailedResolveUrls = new HashSet<>();
   }
 
   /**
    * Add a UrlDevice to the collection.
    * @param urlDevice The UrlDevice to add.
+   * @return alreadyFound Had the device already existed in the map
```

Comment:
@return doesn't need a variable name as part of the description, so I'd reword this as "@return true if the device already existed in the map" (or similar)